src/operators.jl

```diff
@@ -20,8 +20,12 @@ jointerms(terms1::AbstractArray{<:Term}, terms2::AbstractArray{<:Term}) = merges
 
 # Multiplication is handled as a special case so that we can write these
 # definitions without resorting to promotion:
-MP.multconstant(α, v::MonomialLike) = Term(α, Monomial(v))
-MP.multconstant(v::MonomialLike, α) = Term(α, Monomial(v))
+MP.multconstant(v::Monomial, α) = Term(α, v)
+MP.multconstant(v::Variable, α) = Term(α, Monomial(v))
```

Comment:
You should be able to get rid of these two lines thanks to this line:
https://github.com/JuliaAlgebra/MultivariatePolynomials.jl/blob/82cbdcdce39436e075907d6c520f12614e97f8d2/src/operators.jl#L47

Reply:
Ok, thanks! #40